pyproject.toml

```diff
@@ -1,10 +1,64 @@
 [build-system]
 requires = [
-    "setuptools>=56,!=61.0.0",
-    "setuptools_scm[toml]>=6.2",
-    "wheel"
-    ]
-build-backend = 'setuptools.build_meta'
+  "setuptools>=56,!=61.0.0",
+  "setuptools_scm[toml]>=6.2",
+  "wheel",
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "sunkit_spex"
+description = "A package for solar X-ray spectroscopy."
+requires-python = ">=3.9"
+readme = { file = "README.rst", content-type = "text/x-rst" }
+license = { file = "licenses/LICENSE.rst" }
+authors = [
+  { name = "The SunPy Community", email = "sunpy@googlegroups.com" },
+]
+dependencies = [
```

Comment:
Are all these dependences needed in the required list?

Reply:
See #164